Sources/PortalSwift/Core/Mpc/PortalMpc.swift

```diff
@@ -10,8 +10,30 @@ import Foundation
 import Mpc
 import Security
 
+public protocol PortalMpcProtocol {
```

Comment:
can we put this in a separate file? similar to how we have the MpcMobileProtocol.